src/hypothesis/_settings.py

```diff
@@ -619,15 +605,6 @@ def _validate_phases(phases):
 )
 
 settings.define_setting(
-    'report_statistics',
-    default=Statistics.interesting,
-    description=u"""
-If set to True, Hypothesis will run a preliminary health check before
```

Comment:
Same story as for `max_mutations`: no changelog mention and removal without deprecation.

Reply:
Ok, I miss the top post about (: Sorry, nvm, ok here.

Reply:
No worries, this kind of sanity-check is why we do reviews 😄